### PR TITLE
Converted quests actions to revscriptsys

### PR DESCRIPTION
**Pull request by heberholanda**

### CODE REVIEW

File: data/scripts/actions/quests/fathers_burden/corpse.lua [hunk 1]
```diff
@@ -3,6 +3,7 @@ local config = {
 	[12546] = {itemId = 12504, storage = Storage.FathersBurden.Corpse.Sinew, text = 'Heoni\'s sinew'}
 }
 
+local fathers = Action()
 function onUse(player, item, fromPosition, target, toPosition, isHotkey)
```
Comment (majestyotbr) [inline data/scripts/actions/quests/fathers_burden/corpse.lua]: function fathers.onUse(player, item, fromPosition, target, toPosition, isHotkey)

File: data/scripts/actions/quests/green_djinn/tear.lua [hunk 2]
```diff
@@ -1,3 +1,4 @@
+local greenTear = Action()
 function onUse(player, item, fromPosition, target, toPosition, isHotkey)
```
Comment (majestyotbr) [inline data/scripts/actions/quests/green_djinn/tear.lua]: function greenTear.onUse(player, item, fromPosition, target, toPosition, isHotkey)

File: data/scripts/actions/quests/tinder_box/tinder_box.lua [hunk 3]
```diff
@@ -1,11 +1,9 @@
--- <action itemid="22728" script="tinderBox.lua" />
-
 local config = {
 	item = 22728,
 	target = 22727,
 	reward = 22726,
 }
-
+local tinder_box = Action()
 function onUse(player, item, fromPosition, target, toPosition, isHotkey)
```
Comment (majestyotbr) [inline data/scripts/actions/quests/tinder_box/tinder_box.lua]: function tinder_box.onUse(player, item, fromPosition, target, toPosition, isHotkey)

**Review by heberholanda — COMMENTED** [review]
ok

**Review by majestyotbr — APPROVED** [review]

**Pull request merged**